proto/backup.proto

```diff
@@ -124,6 +125,26 @@ message S3 {
     string secret_access_key = 9;
 }
 
+// GCS storage backend saves files into google cloud storage.
+message GCS {
+    string endpoint = 1;
```

Comment:
I think no endpoint for gcs.

Reply:
See https://godoc.org/google.golang.org/api/option#WithEndpoint